### PR TITLE
Remove unnecessary null check.

### PR DESCRIPTION
**Pull request by liaozan**
Remove unnecessary null check.

### CODE REVIEW

**Comment by ralf0131** [comment]
Hi, please do not change the indent from space to tab.

**Comment by liaozan** [comment]
updated

File: dubbo-config/dubbo-config-spring/src/main/java/com/alibaba/dubbo/config/spring/AnnotationBean.java [hunk 1]
```diff
@@ -263,40 +262,41 @@ private Object refer(Reference reference, Class<?> referenceClass) { //method.ge
...
-                    && "".equals(reference.interfaceName())
-                    && referenceClass.isInterface()) {
+                && "".equals(reference.interfaceName())
+                && referenceClass.isInterface()) {
                 referenceConfig.setInterface(referenceClass);
             }
             if (applicationContext != null) {
                 referenceConfig.setApplicationContext(applicationContext);
-                if (reference.registry() != null && reference.registry().length > 0) {
+                reference.registry();
```
[the remark this answers is not in the document]
Comment (liaozan) [reply dubbo-config/dubbo-config-spring/src/main/java/com/alibaba/dubbo/config/spring/AnnotationBean.java]: oh,I am too careless... updated

**Comment by ralf0131** [comment]
Everything is all right, thanks for the pull request.

**Pull request merged**